contracts/mapping-token/contracts/v2/Backing.sol

```diff
@@ -0,0 +1,248 @@
+
+    struct LockedInfo {
+        address token;
+        address sender;
+        uint256 amount;
+    }
+    struct InBoundLaneInfo {
+        address remoteSender;
+        address inBoundLaneAddress;
+    }
+    uint32 public constant NATIVE_TOKEN_TYPE = 0;
+    uint32 public constant ERC20_TOKEN_TYPE = 1;
+    address public feeMarket;
+    string public thisChainName;
+    uint32 public remoteChainPosition;
+    address public remoteMappingTokenFactory;
+
+    // bridge channel
+    // bridgedLanePosition => inBoundLaneAddress
+    mapping(uint32 => address) public inboundLanes;
+    // bridgedLanePosition => outBoundLaneAddress
+    mapping(uint32 => address) public outboundLanes;
+
+    // tokenAddress => reistered
+    mapping(address => bool) public registeredTokens;
+
+    // (messageId => tokenAddress)
+    mapping(uint256 => address) public registerMessages;
+    // (messageId => lockedInfo)
+    mapping(uint256 => LockedInfo) public lockMessages;
+
+    event NewInBoundLaneAdded(address backingAddress, address inboundLane);
+    event NewOutBoundLaneAdded(uint32 bridgedLanePosition, address outboundLane);
+    event NewErc20TokenRegistered(uint256 messageId, address token);
+    event TokenLocked(uint256 messageId, address token, address recipient, uint256 amount);
+    event TokenLockFinished(uint256 messageId, bool result);
+    event TokenRegisterFinished(uint256 messageId, bool result);
+    event TokenUnlocked(address token, address recipient, uint256 amount);
+
+    modifier onlyInBoundLane(address mappingTokenFactoryAddress) {
+        (,,uint32 bridgedChainPosition, uint32 bridgedLanePosition) = IMessageCommitment(msg.sender).getLaneInfo();
+        require(remoteChainPosition == bridgedChainPosition, "Backing:Invalid bridged chain position");
+        require(remoteMappingTokenFactory == mappingTokenFactoryAddress, "Backing:remote caller is not issuing account");
+        require(inboundLanes[bridgedLanePosition] == msg.sender, "Backing:caller is not the inboundLane account");
+        _;
+    }
+
+    modifier onlyOutBoundLane() {
+        (,,uint32 bridgedChainPosition, uint32 bridgedLanePosition) = IMessageCommitment(msg.sender).getLaneInfo();
+        require(remoteChainPosition == bridgedChainPosition, "Backing:Invalid bridged chain position");
+        require(outboundLanes[bridgedLanePosition] == msg.sender, "Backing:caller is not the outboundLane account");
+        _;
+    }
+
+    function initialize(uint32 _bridgedChainPosition, address _remoteMappingTokenFactory, address _feeMarket, string memory _chainName) public initializer {
+        feeMarket = _feeMarket;
+        remoteChainPosition = _bridgedChainPosition;
+        remoteMappingTokenFactory = _remoteMappingTokenFactory;
+        thisChainName = _chainName;
+        ownableConstructor();
+    }
+
+    function unpause() external onlyOwner {
+        _unpause();
+    }
+
+    function pause() external onlyOwner {
+        _pause();
+    }
```

Comment:
`owner` will be `TimeLock`, and if an emergency occur, we could not `pause` Immediately.
So the owner could be split to `guard team` and `configue team`. The same to `mappingTokenFactory`
@HackFisher @xiaoch05

Reply:
good catch, we need an operator account to do this pause/unpause operation. And this account is managed by Owner(Dao)